js/os.ts

```diff
@@ -336,3 +336,29 @@ export function writeFileSync(
   const msg = fbs.WriteFileSync.endWriteFileSync(builder);
   send(builder, fbs.Any.WriteFileSync, msg);
 }
+
+/**
+ * Changes the permissions of file or folder
+ *     import { chmodSync } from "deno";
+ *     const path = 'path';
+ *     const mode = '755'; // octal number in string
+ *
+ *     chmodSync(path, mode);
+ */
+export function chmodSync(path: string, mode: string): void {
```

Comment:
I think we should use an integer for the `mode` - as Node does - https://nodejs.org/api/fs.html#fs_file_modes